Sensors/main_user.py

```diff
@@ -0,0 +1,110 @@
+#!usr/bin/env python3
+#-*-coding:utf-8-*-
+
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning) #in order to avoid warning for each http request
+from httpRequester import getSettings, postSettings
+
+from settings import Settings
+
+def help():
+    print("Commands :")
+    print("-'get' to get the current edge settings")
+    print("-'help' or 'h' for the list of commands")
+    print("-'load' to save in local the current edge settings")
+    print("-'new' to create a new settings with default value")
+    print("-'new <name> <clim_on> <clim_off> <heat_on> <heat_on> <temperature_unit>' to create a new settings in local")
+    print("-'post' to post your local edge settings if it exists")
+    print("-'quit' or 'q' to quit application")
+    print("-'see' to see the current settings in local")
+    print("-'set <element> <value>' to set in local one of the values, type 'set -h' for details")
+
+def helpSet():
+    print("'set <element> <value>' :")
+    print("-<element> is either 'name', 'clim_off', 'clim_on', 'heat_off', 'heat_on' or 'unit'")
+    print("-if <element> is clim_off/on or heat_off/on, <value> must be a real number")
+    print("-if <element> is 'unit', value is either 'C' or 'F'")
+
+def load():
+    r = getSettings()
+    name = r.json().get("house_name")
+    cOn = r.json().get("clim_on")
+    cOff = r.json().get("clim_off")
+    hOn = r.json().get("heat_on")
+    hOff = r.json().get("heat_off")
+    tUnit = r.json().get("temperature_unit")
+    return Settings(name, cOn, cOff, hOn, hOff, tUnit)
```

Comment:
Should be integrated as a constructor or class method of Settings class directly